sourcetracker/_cli/gibbs.py

```diff
@@ -180,11 +212,30 @@ def gibbs(table_fp: Table,
     mpm.to_csv(os.path.join(output_dir, 'mixing_proportions.txt'), sep='\t')
     mps.to_csv(os.path.join(output_dir, 'mixing_proportions_stds.txt'),
                sep='\t')
-
+    # need to count number of rows here to check for equality
+    color_list = bar_color.split()
     # Plot contributions.
-    fig, ax = plot_heatmap(mpm.T)
-    fig.savefig(os.path.join(output_dir, 'mixing_proportions.pdf'), dpi=300)
-
+    graphs = ST_graphs(mpm, output_dir, title=title, color=heatmap_color)
+    if heatmap:
+        graphs.ST_heatmap()
+        if not unknowns:
+            graphs.ST_heatmap(unknowns=False)
+    if paired_heatmap:
+        graphs.ST_paired_heatmap()
+        if not unknowns:
+            graphs.ST_paired_heatmap(unknowns=False)
+            graphs.ST_paired_heatmap(unknowns=False, normalized=True)
+        if transpose:
+            graphs.ST_paired_heatmap(unknowns=False, normalized=True,
+                                     transpose=True)
+            graphs.ST_paired_heatmap(normalized=True,
+                                     transpose=True)
+            graphs.ST_paired_heatmap(transpose=True)
```

Comment:
To reduce reduce repeat code please remove set variables in branches and call ST_paired_heatmap once.